Makefile

```diff
@@ -1,3 +1,5 @@
+DC_RUN ?= docker-compose run --rm
```

Comment:
To avoid issue like this in the futur I add this variable for all `docker-compose run` command. WDYT?

Reply:
Sounds good to me! :-)